paasta_tools/paasta_remote_run.py

```diff
@@ -37,18 +44,34 @@
 
 def parse_args(argv):
     parser = argparse.ArgumentParser(description='')
-    add_remote_run_args(parser)
-    parser.add_argument(
+    subs = parser.add_subparsers(dest='action', help='Subcommands of paasta_remote_run')
+
+    start_parser = subs.add_parser('start', help='Start task')
+    add_start_args_to_parser(start_parser)
+    add_common_args_to_parser(start_parser)
+    start_parser.add_argument(
         '-X', '--constraints-json',
         help=('Mesos constraints JSON'),
         required=False,
         default=None,
     )
+
+    stop_parser = subs.add_parser('stop', help='Stop task')
+    add_common_args_to_parser(stop_parser)
+    stop_parser.add_argument(
+        '-F', '--framework-id',
```

Comment:
there are 2 ways to uniquely identify remote-run: compound identifier of (service, cluster, instance, run-id) and framework-id. i only decided to expose framework-id for completeness purpose, i don't expect people to use it very often.

we also can't just use instance. normally i don't expect users to even use instance, it will default to "remote" and is just a configuration preset. potentially people would have multiple instances for some wildly different batch jobs, but the problem i was trying to solve with run-id is to identify a remote-run among multiple with the same (service, cluster, instance) value.

we can't force users to just have different instances, because it kind of defeats the purpose of ad hoc runs.

we can implement a chooser for when user didn't provide neither run-id nor framework id, it would be useful.